numprocs=8

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v0_batch0/v0_batch0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py

Start time:  2021-02-25 20:04:32.781022
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py
Importing netParams from ../data/v0_batch0/v0_batch0_netParams.py

Creating network of 14 cell populations on 8 hosts...
  Number of cells on node 5: 48 
  Number of cells on node 6: 47 
  Number of cells on node 7: 47 
  Number of cells on node 4: 48 
  Number of cells on node 2: 48 
  Number of cells on node 1: 48 
  Number of cells on node 3: 48 
  Number of cells on node 0: 48 
  Done; cell creation time = 1.39 s.
Making connections...
  Number of connections on node 3: 333 
  Number of synaptic contacts on node 3: 8638 
  Number of connections on node 6: 326 
  Number of synaptic contacts on node 6: 8742 
  Number of connections on node 4: 334 
  Number of synaptic contacts on node 4: 8950 
  Number of connections on node 2: 358 
  Number of synaptic contacts on node 2: 9482 
  Number of connections on node 7: 371 
  Number of synaptic contacts on node 7: 9734 
  Number of connections on node 0: 376 
  Number of synaptic contacts on node 0: 9918 
  Number of connections on node 5: 361 
  Number of synaptic contacts on node 5: 9380 
  Number of connections on node 1: 373 
  Number of synaptic contacts on node 1: 9924 
  Done; cell connection time = 2.72 s.
Adding stims...
  Number of stims on node 7: 94 
  Number of stims on node 0: 96 
  Number of stims on node 1: 96 
  Number of stims on node 5: 96 
  Number of stims on node 2: 96 
  Number of stims on node 6: 94 
  Number of stims on node 3: 96 
  Number of stims on node 4: 96 
  Done; cell stims creation time = 0.02 s.
Recording 1 traces of 1 types on node 6
Recording 1 traces of 1 types on node 1
Recording 1 traces of 1 types on node 7
Recording 2 traces of 1 types on node 0
Recording 2 traces of 1 types on node 5
Recording 2 traces of 1 types on node 2
Recording 2 traces of 1 types on node 3
Recording 3 traces of 1 types on node 4
0.0s

Running simulation for 2000.0 ms...
0.1s
0.2s
0.3s
0.4s
0.5s
0.6s
0.7s
0.8s
0.9s
1.0s
1.1s
1.2s
1.3s
1.4s
1.5s
1.6s
1.7s
1.8s
1.9s
2.0s
  Done; run time = 777.27 s; real-time ratio: 0.00.

Gathering data...
  Done; gather time = 1.12 s.

Analyzing...
Plotting connectivity matrix...
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa88fe10>)
Plotting connectivity matrix...
Plotting connectivity matrix...
Plotting 2D representation of network cell locations and connections...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc2870a550>)
Plotting connectivity matrix...
Plotting 2D representation of network cell locations and connections...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f15301068c0>)
Plotting connectivity matrix...
Plotting 2D representation of network cell locations and connections...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5708f7d0>)Plotting connectivity matrix...

Plotting 2D representation of network cell locations and connections...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08b84050>)
Plotting 2D representation of network cell locations and connections...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f67332bf870>)  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487a4a50>)
Plotting 2D representation of network cell locations and connections...

Plotting 2D representation of network cell locations and connections...
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc286ff7d0>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa5ac1e0>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f1530106280>)
Plotting 2D representation of network cell locations and connections...
Plotting 2D representation of network cell locations and connections...
Plotting 2D representation of network cell locations and connections...
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5708f230>)
Plotting 2D representation of network cell locations and connections...
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08a290a0>)
Plotting 2D representation of network cell locations and connections...
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f67332bf140>)
Plotting 2D representation of network cell locations and connections...
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487a44b0>)
Plotting 2D representation of network cell locations and connections...
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa5fe3c0>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f152fe3fd20>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc286f69b0>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5702dfa0>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08e28fa0>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f6732f89780>)
  There was an exception in plot2Dnet(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487a8910>)
  Cells: 382
  Connections: 2832 (7.41 per cell)
  Synaptic contacts: 74768 (195.73 per cell)
  Spikes: 452 (0.59 Hz)
  Simulated time: 2.0 s; 8 workers
  Run time: 777.27 s
   L1_DAC : 0.500 Hz
   L1_DLAC : 0.200 Hz
   L1_HAC : 1.263 Hz
   L1_NGC-DA : 0.333 Hz
   L1_NGC-SA : 0.682 Hz
   L1_SLAC : 2.056 Hz
   L23_BP : 3.583 Hz
   L23_BTC : 0.682 Hz
   L23_ChC : 0.192 Hz
   L23_DBC : 1.595 Hz
   L23_LBC : 0.405 Hz
   L23_MC : 0.257 Hz
   L23_NBC : 0.170 Hz
   L23_NGC : 0.000 Hz
Plotting recorded cell traces ... cell
Plotting recorded cell traces ... cell
Plotting recorded cell traces ... cell
Plotting recorded cell traces ... cell
Plotting recorded cell traces ... cell
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f152fe3f410>)
Plotting recorded cell traces ... cell
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa5f8c30>)
Plotting recorded cell traces ... cell
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08e2e780>)
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487abb90>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487abc30>)  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc286f9fa0>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc286f9730>)Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f152fe3fc80>)
Plotting recorded cell traces ... trace
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f6732d0b4b0>)
Plotting raster...
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa5f8b90>)
Plotting recorded cell traces ... trace
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08e2e820>)
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5702d730>)
Plotting raster...


Plotting recorded cell traces ... trace
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa5f8b40>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5702dfa0>)
Plotting recorded cell traces ... trace
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f152fe3f500>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f6732d0b550>)
Plotting recorded cell traces ... trace
Plotting recorded cell traces ... trace
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f6732d0b5a0>)
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08e2e870>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f61aa5f8b90>)
Plotting recorded cell traces ... trace
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487ab960>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f152fe3fc80>)
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc286f9f00>)
  There was an exception in plotTraces(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5702d820>)
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f5e487abc30>)
Plotting raster...
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7fef08e2e820>)
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7efc286f9730>)
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f6732d0b550>)
Plotting raster...
  There was an exception in plotRaster(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'"), <traceback object at 0x7f9f5702dfa0>)
Saving output as ../data/v0_batch0/v0_batch0_0.json  ... 
Finished saving!
  Done; saving time = 0.72 s.
Plotting connectivity matrix...
Saving figure data as ../data/v0_batch0/v0_batch0_0_pop_numConns_matrix.json ... 
Plotting 2D representation of network cell locations and connections...
Saving figure data as ../data/v0_batch0/v0_batch0_0_xy.json ... 
Plotting 2D representation of network cell locations and connections...
Saving figure data as ../data/v0_batch0/v0_batch0_0_xz.json ... 
Plotting recorded cell traces ... cell
Plotting raster...
Plotting recorded cell traces ... trace
Plotting raster...
